src/main/java/picard/sam/ValidateSamFile.java

```diff
@@ -210,6 +217,9 @@ protected int doWork() {
             } else {
                 validator.setVerbose(true, MAX_OUTPUT);
             }
+            if (SKIP_MATE_VALIDATION) {
+                validator.setSkipMateValidation(SKIP_MATE_VALIDATION);
```

Comment:
I realize that this is the usual way this is done, but I don't see why the `if` is necessary here.

Reply:
Do you want me to change it here?  You are the reviewer....